### PR TITLE
better related content designs

### PR DESCRIPTION
**Pull request by webb04**
## Screenshots

| Before | After |
| --- | --- |
|  |  |
|  |  |

### CODE REVIEW

File: src/renderer.ts [hunk 1]
```diff
@@ -313,6 +313,23 @@ const richLinkWidth = '8.75rem';
...
+        margin-bottom: 0;
+        padding-bottom: 0;
+    }
+
+    p::before {
+        ${icons}
+        content: '\\e005';
+        border-radius: 100%;
+        border: solid 1px ${neutral[7]};
+        padding: 4px 5px 5px 5px;
```
Comment (webb04) [inline src/renderer.ts:329]: The icon font has additional space above the character. Hopefully we can solve this if we move to svgs.

File: src/renderer.ts [hunk 2]
```diff
@@ -313,6 +313,23 @@ const richLinkWidth = '8.75rem';
...
+        font-size: 12px;
```
Comment (JamieB-gu) [inline src/renderer.ts:330]: We might want these in `rem` to avoid scaling issues?

File: src/renderer.ts [hunk 3]
```diff
@@ -346,7 +367,7 @@ const richLinkStyles = css`
 
 const RichLink = (props: { url: string; linkText: string }): ReactElement =>
     styledH('aside', { css: richLinkStyles },
-        styledH('a', { href: props.url }, [h('h1', null, props.linkText), 'Read more'])
+        styledH('a', { href: props.url }, [h('h1', null, props.linkText), h('p', null, 'Read more')])
```
Comment (JamieB-gu) [inline src/renderer.ts:370]: I know this is wrapped in a link, but should we make this a `<button>` semantically?

File: src/renderer.ts (hunk 2 above)
Comment (webb04) [reply src/renderer.ts:330]: I'm a bit worried about the circle looking less round at some sizes, I think we can definitely use rem sizing for the svg though https://github.com/guardian/source/issues/371

**Pull request merged**